angular.json

```diff
@@ -327,41 +327,40 @@
           }
         }
       }
-    }
-  },
-  "cds": {
-    "root": "projects/cds",
-    "sourceRoot": "projects/cds/src",
-    "projectType": "library",
-    "prefix": "cx",
-    "architect": {
-      "build": {
-        "builder": "@angular-devkit/build-ng-packagr:build",
-        "options": {
-          "tsConfig": "projects/cds/tsconfig.lib.json",
-          "project": "projects/cds/ng-package.json"
-        }
-      },
-      "test": {
-        "builder": "@angular-devkit/build-angular:karma",
-        "options": {
-          "main": "projects/cds/src/test.ts",
-          "tsConfig": "projects/cds/tsconfig.spec.json",
-          "karmaConfig": "projects/cds/karma.conf.js"
-        }
-      },
-      "lint": {
-        "builder": "@angular-devkit/build-angular:tslint",
-        "options": {
-          "tsConfig": [
-            "projects/cds/tsconfig.lib.json",
-            "projects/cds/tsconfig.spec.json"
-          ],
-          "exclude": ["**/node_modules/**"]
+    },
```

Comment:
Did you change something in this file? Or is this just a formatting change?

Reply:
I changed it on the epic branch as well when I merged. I will check out this branch from the epic so that this diff doesn't show up

Reply:
Ok, let me know when it's done.